docker/standalone/Dockerfile

```diff
@@ -26,18 +21,19 @@ COPY ./docker/standalone/dashboard-entrypoint.sh /lh
 COPY ./docker/standalone/docker-entrypoint.sh /lh
 COPY ./docker/standalone/log4j2.properties /lh
 
-COPY ./dashboard /lh/dashboard
 WORKDIR /lh/dashboard
 ENV NODE_ENV=production
 EXPOSE 8080
 
 COPY ./dashboard/apps/web/.next/standalone ./
 COPY ./dashboard/apps/web/.next/static ./apps/web/.next/static
 COPY ./dashboard/apps/web/public ./apps/web/public
-COPY ./docker/dashboard/docker-entrypoint.sh ./
 
 WORKDIR /
 
-COPY --from=builder /lh/server/build/libs/server-*-all.jar /lh/server.jar
+COPY ./server/build/libs/server-*-all.jar /lh/server.jar
+
+ENV LHD_API_HOST=localhost
```

Comment:
This environment variables where missing for the dashboard to work inside `lh-standalone`